doc/manual/config-advanced.rst

```diff
@@ -346,6 +346,10 @@ Although each judgedaemon process will be bound to one single CPU
 core, shared use of other resources such as disk I/O might
 still have effect on run times.
 
+Real world usage shows that the amount of judgedaemons a machine can run
+does not scale with the number of cores. As a general advice verify the
+timelimits again when more active judgedaemons are added.
```

Comment:
I think this statement requires some more explanation. The problem is not that it doesn't scale, but that while scaling to multiple judgedaemons, runtimes become less consistent. That's also already partially mentioned in the previous paragraph, so I think it's better to either leave this out or merge it in with that, and e.g. mention that performance with mulitple judgedaemons can fluctuate up to 20-40%.

Reply:
I've removed the section.